<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preference</class>
 <widget class="QDialog" name="Preference">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preference</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="waylandBackendLabel">
         <property name="text">
          <string>Wayland Backend</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="waylandBackendComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="xorgBackendLabel">
         <property name="text">
          <string>Xorg Backend</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="xorgBackendComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="autoWindowSystemCheckBox">
       <property name="text">
        <string>Automatically Change Window System ( xorg / wayland )</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="autoChangeBackendCheckBox">
       <property name="text">
        <string>Automatically Change Backend</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="bottomSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Preference</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Preference</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Preference</sender>
   <signal>finished(int)</signal>
   <receiver>Preference</receiver>
   <slot>saveConfig(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveConfig(int)</slot>
 </slots>
</ui>
